.gitlab-ci.yml

```diff
@@ -7,15 +7,9 @@
   before_script:
    - echo "$CI_JOB_TOKEN" | docker login -u gitlab-ci-token $CI_REGISTRY --password-stdin
   script:
-    # Download test shim.
-    - wget -O ${PWD}/tests/test.sh https://gitlab.com/snippets/1751673/raw
-    - chmod +x ${PWD}/tests/test.sh
+    # run ci tests
+    - ${PWD}/tests/runtests.sh
 
-    # Run tests.
-    - ${PWD}/tests/test.sh
-
-    # Make sure OpenWISP is running.
-    - ./tests/runtests.sh
   dependencies: []
```

Comment:
I am not sure if this line serves any purpose, does it?

Reply:
It's use to to disable artifact passing for this job.

For more details see https://docs.gitlab.com/ce/ci/yaml/